.github/workflows/branch.yml

```
@@ -0,0 +1,135 @@
+name: Kirby branch CI
+on:
+  push:
+    branches-ignore: [ master ]
+defaults:
+  run:
+    shell: bash
+
+jobs:
+    lint_and_unittest:
+      runs-on: ubuntu-latest
+      strategy:
+       matrix:
+          node-version: [10.16]
+      steps:
+        - uses: actions/checkout@v2
+        - name: Use Node.js ${{ matrix.node-version }}
+          uses: actions/setup-node@v1
+          with:
+            node-version: ${{ matrix.node-version }}
```

Why this? Couldn't it simplified to:
```suggestion
      env:
       node-version: [10.16]
      steps:
        - uses: actions/checkout@v2
        - name: Use Node.js ${{ env.node-version }}
          uses: actions/setup-node@v1
          with:
            node-version: ${{ env.node-version }}
```